src/core/utils.ts

```diff
@@ -113,7 +113,13 @@ export function reverseDeepMerge(dest, source = undefined) {
         if (isFunction(src[srcArg])) {
           dest[srcArg] = src[srcArg];
         } else {
-          dest[srcArg] = clone(src[srcArg]);
+          try {
```

Comment:
@ericof can you provide me a reproducing example with a property which is not used on the form I'm pretty sure we can find a better way